Formula/mosl.rb

```diff
@@ -0,0 +1,19 @@
+class Mosl < Formula
+  desc "Bash script to audit and fix macOS Mojave (10.14.x) security settings"
+  homepage "https://github.com/0xmachos/mOSL"
+  url "https://github.com/0xmachos/mOSL/archive/v2.4.0.tar.gz"
+  sha256 "6cf2f301844085f36aa131867a4cc11d7d11a3b04b25c5d3ea35f71c39296eef"
+
+  depends_on "minisign" => :recommended
+  # Required to valide signature against Lockdown.minisig
+
+  def install
+    bin.install "Lockdown"
+    prefix.install "Lockdown.minisig"
+    # Signature file generated by minisign
+  end
+
+  test do
+    assert_equal version.to_s, shell_output("#{bin}/Lockdown debug | grep 'mOSL Version' | awk '{print $3}' | tr -d 'v'").chomp
```

Comment:
We need a better test here:

1. Tests shouldn't pipe out to awk or other unix tools to do text processing
2. The test should actually test some sort of functionality or correctness of the installation